infer/modules/train/extract/extract_f0_print.py

```diff
@@ -16,19 +16,84 @@
 logging.getLogger("numba").setLevel(logging.WARNING)
 from multiprocessing import Process
 
-exp_dir = sys.argv[1]
-f = open("%s/extract_f0_feature.log" % exp_dir, "a+")
+# exp_dir = sys.argv[1]
+# f = open("%s/extract_f0_feature.log" % exp_dir, "a+")
 
 
 def printt(strr):
     print(strr)
-    f.write("%s\n" % strr)
-    f.flush()
 
+class DMLFeatureInputRMVPE(object):

```

Comment:
As for the f0s, you can use the classes in rvc.f0. I have almost finished them. Meanwhile, be careful to compare the difference between rvc.f0 and code in the training.

Reply:
> As for the f0s, you can use the classes in rvc.f0. I have almost finished them. Meanwhile, be careful to compare the difference between rvc.f0 and code in the training.

Speaking of which, shouldn't the `rvc/f0/__init__.py` have
```
__all__ = ["F0Predictor", "CRePE", "Dio", "Harvest", "PM", "RMVPE"]
```
In it?

Reply:
Still WIP. You can open another PR.